.github/workflows/pr-quality-check.yml

```
@@ -0,0 +1,450 @@
+    if: github.event.pull_request.draft == false
+
+    steps:
+      - name: 📥 Checkout PR
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: 🔍 PR Information
+        id: pr_info
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)